src/hooks/useChat.ts

```diff
@@ -16,8 +16,6 @@ import {
 } from '../utils/arrayHelpers';
 import { usePrevious } from './usePrevious';
 
-const INITIAL_MESSAGES_COUNT = 50;
```

Comment:
[Default page size](https://www.twilio.com/docs/usage/twilios-response#response-formats-list-filters) is 50 anyway